chocolatey/datahelix/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,84 @@
+﻿# IMPORTANT: Before releasing this package, copy/paste the next 2 lines into PowerShell to remove all comments from this file:
+#   $f='c:\path\to\thisFile.ps1'
+#   gc $f | ? {$_ -notmatch "^\s*#"} | % {$_ -replace '(^.*?)\s*?[^``]#.*','$1'} | Out-File $f+".~" -en utf8; mv -fo $f+".~" $f
+
+## NOTE: In 80-90% of the cases (95% with licensed versions due to Package Synchronizer and other enhancements),
+## AutoUninstaller should be able to detect and handle registry uninstalls without a chocolateyUninstall.ps1.
+## See https://chocolatey.org/docs/commands-uninstall
+## and https://chocolatey.org/docs/helpers-uninstall-chocolatey-package
+
+## If this is an MSI, ensure 'softwareName' is appropriate, then clean up comments and you are done.
+## If this is an exe, change fileType, silentArgs, and validExitCodes
+
+$ErrorActionPreference = 'Stop'; # stop on all errors
+$packageArgs = @{
+  packageName   = $env:ChocolateyPackageName
+  softwareName  = 'datahelix*'  #part or all of the Display Name as you see it in Programs and Features. It should be enough to be unique
+  fileType      = 'EXE_MSI_OR_MSU' #only one of these: MSI or EXE (ignore MSU for now)
```

Comment:
didn't think we had any of these file types?

Reply:
These are the only options for a chocolatey package, it might be possible to omit the setting tho.